Sources/Clappr/Classes/Base/Core.swift

```diff
@@ -176,6 +176,7 @@ open class Core: UIObject, UIGestureRecognizerDelegate {
 
     fileprivate func loadSourceIfNeeded() {
         if let source = options[kSourceUrl] as? String {
+            trigger(.willLoadSource)
```

Comment:
Recommend adding a `load` method to `Core` to separate it from options change. `load` should not be an options update side-effect.

Reply:
Done.